conf/etc/ha_stop_io

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+
+# Extract keys matching to STR from consul that contain the string stop_io
+STR="cortx/base/ha/obj/node/"
+# search in consul for "action": "stop_io"
+OUTPUT=$( consul kv get -recurse $STR | grep "\"action\": \"stop_io\"" )
+
+
+if [ "$OUTPUT" ]
+then
+    #execute resource stop
+    pcs resource disable lnet-c1 lnet-c2
```

Comment:
Add logs
Please add log to know that it has be triggered

Reply:
Logging pcs status immediately after resource disable may give partial status.
i.e. some services may be stopped and some may be in the process of stopping.

Reply:
done